src/components/__tests__/Trophy.spec.tsx

```diff
@@ -0,0 +1,25 @@
+import { render } from '@testing-library/react-native'
+import React from 'react'
+
+import Trophy from '../Trophy'
+
+describe('Trophy', () => {
+  it('should show no trophy when level is 0', () => {
+    const { queryByTestId } = render(<Trophy level={0} />)
+    expect(queryByTestId('trophy-0')).toBeNull()
+  })
+
+  it('should show one trophy when level is 1', () => {
+    const { queryByTestId } = render(<Trophy level={0} />)
```

Comment:
```suggestion
    const { queryByTestId } = render(<Trophy level={1} />)
```

Reply:
oke null is also defined, therefore the test were not failing and completely useless. Fixed it.